lib/inferno/entities/test.rb

```diff
@@ -106,11 +106,13 @@ class << self
         # @return [void]
         # @example
         #   input :patient_id, :bearer_token
-        def input(*input_definitions)
+        def input(name, *other_names, **input_definition)
```

Comment:
The documentation here and in runnable will need to be updated.

Reply:
Updated.

Reply:
The individual options should be specified: https://rubydoc.info/gems/yard/file/docs/Tags.md#option